source/vibe/core/log.d

```diff
@@ -423,9 +423,12 @@ final class FileLogger : Logger {
 
 	override void put(scope const(char)[] text)
 	{
-		static if (__VERSION__ <= 2066)
+		static if (__VERSION__ <= 2066) {
```

Comment:
I'd say we just drop this, we don't support this version anymore...

Reply:
Yeah, I'll remove that.